src/App.css

```diff
@@ -56,6 +56,36 @@
   --grey-medium: #a0aec0;
   --grey-light: #e2e8f0;
   --white: #fff;
+} */
+
+/* Custom Blue */
+* {
```

Comment:
replace `*` with `:root` i think you're applying this rule to every element rather than simply making it globally available.  Not sure if this would cause perf issues, probably not but idk.